docs/glossary.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Glossary
+---
+
+## Terms
+
+### button
+#### See Also
+*   [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)
+
+### component
+A small, reusable piece of code that that returns an element to render.
+#### See Also
+*   [https://reactjs.org/docs/glossary.html#components](https://reactjs.org/docs/glossary.html#components)
+
+### computed property
```

Comment:
I think this definition is a little to Enyo-specific, but maybe we could clarify it?